tests/test_phy_update.py

```diff
@@ -47,56 +47,51 @@
 
 from pc_ble_driver_py.ble_driver import (
     BLEAdvData,
-    BLEUUID,
-    BLEGapPhys,
+    BLEHci,
     driver,
 )
 
 logger = logging.getLogger(__name__)
 
-req_phy = [driver.BLE_GAP_PHY_1MBPS, driver.BLE_GAP_PHY_1MBPS]
-
 
 class Central(BLEDriverObserver, BLEAdapterObserver):
     def __init__(self, tester, adapter):
         self.tester = tester
         self.adapter = adapter
-        logger.info(
-            "Central adapter is %d", self.adapter.driver.rpc_adapter.internal
-        )
+        logger.info(f"Central adapter is {self.adapter.driver.rpc_adapter.internal}")
         self.conn_q = Queue()
-        self.notification_q = Queue()
+        self.phy_q = Queue()
         self.adapter.observer_register(self)
         self.adapter.driver.observer_register(self)
         self.conn_handle = None
         self.phy_rsp = None
         self.new_phy = None
-        
+
     def stop(self):
         if self.conn_handle:
             self.adapter.driver.ble_gap_disconnect(self.conn_handle)
 
-    def start(self, connect_with, req_phy):
+    def start(self, connect_with, req_phys):
         self.connect_with = connect_with
         logger.info(f"scan_start, trying to find {self.connect_with}")
         scan_duration = 5
         self.adapter.driver.ble_gap_scan_start()
         try:
             self.conn_handle = self.conn_q.get(timeout=scan_duration)
-            resp = self.adapter.phy_update(self.conn_handle, req_phy)
-            self.new_phy = resp 
+            logging.info(f"Central requesting: {req_phys=}")
```

Comment:
What does the equal sign mean?

Reply:
The equal sign is a relatively new feature in Python that prints the expression as well as the value.